simple_history/templates/simple_history/_object_history_list.html

```diff
@@ -40,6 +41,14 @@
         <td>
             {{ action.history_change_reason }}
         </td>
+        <td>
+          {% if action.history_diff %}
+            {% for diff in action.history_diff %}
+              {{ diff }} <br>
+            {% endfor %}
+<!--            {{ action.history_diff }}-->
+          {% endif %}
```

Comment:
If my suggestion above is implemented, this can be changed to the following:
```suggestion
          {% if action.history_diff_changes %}
            <ul>
              {% for change in action.history_diff_changes %}
                <li>
                  <code>{{ change.field }}</code>:
                  {{ change.old|truncatechars:100 }}
                  {# Add some spacing, and prevent having the arrow point to the edge of the page if `new` is wrapped #}
                  &nbsp;&rarr;&nbsp;&nbsp;{{ change.new|truncatechars:100 }}
                </li>
              {% endfor %}
            </ul>
          {% endif %}
```
I think the changes would be slightly easier to read if placed inside a `<ul>` list, as suggested above :)

Also note that I added the `truncatechars` filter, in case the old/new values are very long. Not sure if 100 is an "ideal" number of characters, but I think it works.